tests/Tests/500__jupyterhub/test-jupyterlab-git.robot

```diff
@@ -0,0 +1,166 @@
+*** Settings ***
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             OpenShiftCLI
+Library             DebugLibrary
+
+Suite Setup         Server Setup
+Suite Teardown      End Web Test
+
+
+*** Variables ***
+${REPO_URL}         ****
+${DIR_NAME}         Python
+${FILE_PATH}        Python/file.ipynb
+${COMMIT_MSG}       commit msg2
+
+
+*** Test Cases ***
+Verify Pushing Project Changes Remote Repository
+    [Tags]    ODS-326
+    ...       Tier1
+    ${randnum}=    Generate Random String    9    [NUMBERS]
+    ${commit_message}=    Catenate    ${COMMIT_MSG}    ${randnum}
+    Push Some Changes to Repo
+    ...    ${GITHUB_USER.USERNAME}
+    ...    ${GITHUB_USER.TOKEN}
+    ...    ${FILE_PATH}
+    ...    ${REPO_URL}
+    ...    ${commit_message}
+    Clean Up Server
+
+Verify updating your project with changes from a git repository
+    [Tags]    ODS-324
+    ...       Tier1
```

Comment:
Try to make it more readable by moving steps into their own keywords

Reply:
Done